src/styles/utils/icon.scss

```diff
@@ -0,0 +1,27 @@
+.icon {
+  display: block;
+  width: $size-icon;
+  height: $size-icon;
+  background-repeat: no-repeat;
+  transition: 1s;
+  cursor: pointer;
+
+  &__phone {
+    background-image: url(../images/Icon-phone.png);
+  }
+
+  &__menu {
+    background-image: url(../images/Icon-menu.png);
+
+    &:hover {
+      background-image: url(../images/image-menu-hover.png);
+      transition-duration: 1s;
```

Comment:
generally put the transition in the normal state only, as this here makes the menu transition have this timing in the start of the hover but be instant when it's not hovered anymore